init-letsencrypt.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+read -p "Enter your domain name (example.com) : " DOMAIN_NAME
+read -p "Enter Nodejs (frontend) domain (evalai.example.com) : " FRONTEND_DOMAIN
+read -p "Enter Django (backend) domain (evalapi.example.com) : " BACKEND_DOMAIN
+
+domains=(DOMAIN_NAME FRONTEND_DOMAIN BACKEND_DOMAIN)
+
+touch .env
+
+echo "DOMAIN_NAME=$DOMAIN_NAME" > .env
+echo "FRONTEND_DOMAIN=$FRONTEND_DOMAIN" >> .env
+echo "BACKEND_DOMAIN=$BACKEND_DOMAIN" >> .env
+
+rsa_key_size=4096
+data_path="./certbot"
+
+# Adding a valid address is strongly recommended
+read -p "Enter your Email ID : " email
+
+staging=1 # Set to 1 if you're testing your setup to avoid hitting request limits
+
+if [ -d "$data_path" ]; then
+  read -p "Existing data found for $domains. Continue and replace existing certificate? (y/N) " decision
+  if [ "$decision" != "Y" ] && [ "$decision" != "y" ]; then
+    exit
+  fi
+fi
+
+
+if [ ! -e "$data_path/conf/options-ssl-nginx.conf" ] || [ ! -e "$data_path/conf/ssl-dhparams.pem" ]; then
+  echo "### Downloading recommended TLS parameters ..."
+  mkdir -p "$data_path/conf"
+  curl -s https://raw.githubusercontent.com/certbot/certbot/master/certbot-nginx/certbot_nginx/_internal/tls_configs/options-ssl-nginx.conf > "$data_path/conf/options-ssl-nginx.conf"
+  curl -s https://raw.githubusercontent.com/certbot/certbot/master/certbot/certbot/ssl-dhparams.pem > "$data_path/conf/ssl-dhparams.pem"
+fi
+
+echo "### Creating dummy certificate for $domains ..."
+path="/etc/letsencrypt/live/$domains"
```

Comment:
please change this to a fixed path. We want to help the user do minimal changes to deploy on vm. If we can't use ssl certs from shell script parameters then we should ask them to place the certs at a fixed location that will be easier than asking them to make changes to nginx config

Reply:
I don't think that is possible 
Certbot places the certs in a new folder named according to the first domain passed.  
Source : https://certbot.eff.org/docs/using.html#where-are-my-certificates